spec/unit/appmgmt_spec.rb

```diff
@@ -105,76 +215,113 @@ def compile_to_env_catalog(string)
       expect(cons).not_to be_nil
       expect(cons[:consume].ref).to eq("Cap[cap]")
     end
+
+    context "when using a site expression" do
+      it "includes components and capability resources" do
+        require 'debugger'; debugger
+        catalog = compile_to_env_catalog(MANIFEST_WITH_SITE).to_resource
```

Comment:
@lutter This is the test that fails when the resource mentioned above is validated.